tasks/job/run.go

```diff
@@ -113,7 +113,7 @@ func (t *Task) isStale(ctx *context.ExecuteContext) (bool, error) {
 	}
 
 	if len(t.config.Sources.Paths()) != 0 {
-		sourcesLastModified, err := fs.LastModified(t.config.Sources.Paths()...)
+		sourcesLastModified, err := fs.LastModified([]string{}, t.config.Sources.Paths()...)
```

Comment:
It is safe to use a `nil` here instead of allocating space for the `[]string{}`. Iterating over a `nil` slice will do the same thing as an empty slice. I think we can change the first argument in these calls to use `nil`.